engine/src/flutter/impeller/tessellator/tessellator_unittests.cc

```diff
@@ -302,6 +303,82 @@ TEST(TessellatorTest, StrokedCircleTessellationVertices) {
   test(Matrix::MakeScale({0.002, 0.002, 0.0}), {}, 1000.0, 10.0);
 }
 
+TEST(TessellatorTest, FilledArcStripTessellationVertices) {
+  auto tessellator = std::make_shared<Tessellator>();
+
+  auto test = [&tessellator](const Matrix& transform, const Arc& arc) {
+    auto generator = tessellator->FilledArc(transform, arc,
```

Comment:
I think this change is probably best suited for a Playground test. The problem is ensuring that we run it on at least one platform that doesn't support triangle fans. Is that possible?